community/installation-guides/panel/centos7.md

```diff
@@ -41,12 +41,15 @@ We recommend the ius repo to get the latest php packages.
 ```bash
 ## Install Repos
 yum install -y epel-release http://rpms.remirepo.net/enterprise/remi-release-7.rpm
+yum install -y yum-utils
+yum-config-manager --disable remi-php54
+yum-config-manager --enable remi-php73
 
 ## Get yum updates
 yum update -y
 
 ## Install PHP 7.3
-yum install -y php73-php php73-php-common php73-php-fpm php73-php-cli php73-php-json php73-php-mysqlnd php73-php-mcrypt php73-php-gd php73-php-mbstring php73-php-pdo php73-php-zip php73-php-bcmath php73-php-dom php73-php-opcache
+yum install -y php-php php-common php-fpm php-cli php-json php-mysqlnd php-mcrypt php-gd php-mbstring php-pdo php-zip php-bcmath php-dom php-opcache
```

Comment:
`php-php` does not exist. Mistake?

Reply:
Also `php-mcrypt` has been replaced by `php-pecl-mcrypt` btw

Reply:
missed that I had php-php in there.  This worked to install on my test VM I set up still. I assume it's due to php being a require package for the others and remi probably maps php-mcrypt to php-pecl-mcrypt